src/config.rs

```diff
@@ -27,6 +24,12 @@ pub const DEFAULT_UPNP_LEASE_DURATION: Duration = Duration::from_secs(120);
 /// gives 5-6 retries in ~30 s total retry time.
 pub const DEFAULT_INITIAL_RETRY_INTERVAL: Duration = Duration::from_millis(500);
 
+/// Default for [`Config::idle_timeout`] (1 minute).
+///
+/// This is based on average time in which routers would close the UDP mapping to the peer if they
+/// see no conversation between them.
+pub const DEFAULT_IDLE_TIMEOUT: Duration = Duration::from_secs(60);
```

Comment:
This seems long, generally, 20-30 secs is the best we can hope for so setting this at 18 or so would make more sense. Otherwise, we think connections are open (to us) and they are closed. Not much on Inet about timeout, but https://www.quora.com/How-does-UDP-hole-punching-work